src/cms/urls.py

```diff
@@ -938,6 +938,18 @@
                         ]
                     ),
                 ),
+                url(
+                    r"^user_settings/",
+                    include(
+                        [
+                            url(
+                                r"^$",
+                                settings.UserSettingsView.as_view(),
+                                name="user_settings",
+                            ),
+                        ]
+                    ),
```

Comment:
```suggestion
                    r"^user_settings/$",
                    settings.UserSettingsView.as_view(),
                    name="user_settings",
```
Since we only have one url here, we can write this a little bit shorter.